recaf-plugin/src/main/java/dev/xdark/recaf/plugin/SimplePluginManager.java

```diff
@@ -39,7 +38,7 @@ public Collection<PluginLoader> getLoaders() {
 	}
 
 	@Override
-	public Collection<? super Plugin> getPlugins() {
+	public <T extends Plugin> Collection<? super PluginContainer<T>> getPlugins() {
```

Comment:
Some hacky generics, why is it needed?

Reply:
No need now,  I also forgot why it was changed to this

Reply:
Doesn't immediately seem necessary. Compiles and runs without this change.

Reply:
> Doesn't immediately seem necessary. Compiles and runs without this change.

I remember that at the beginning, failed to compile here

Reply:
> nameMap.values() return Collection<PluginContainer<?>>

getPlugins() why not change to public <T extends Plugin> Collection<? super PluginContainer<T>> getPlugins()